### DISCUSSION TITLE
Autoencoder with edge attributes

### DISCUSSION THREAD

**Discussion by MehdiZouitine** [pyg-team/pytorch_geometric#5160] [Q&A]
Hello ! 
I would like to use a GAE (graph auto encoder) to learn a compressed representation of a graph with edge features.
The decoder must be able to reconstruct the attributes of the nodes and the attributes of the edges. 

All the graphs I have are complete graphs (there is an edge between each pair of nodes)
Would you have an architecture to propose to me?

Thank you very much

**Comment by rusty1s** [comment]
I think the [GraphVAE](https://arxiv.org/pdf/1802.03480.pdf) paper is actually what you are looking for.
